From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: laiguo-liang <2909269677@qq.com>
Date: Mon, 20 Jul 2026 00:00:00 +0800
Subject: [PATCH] net-delayacct: add module to net/core/Makefile

Wire net-delayacct.o into the build when CONFIG_NET_DELAYACCT is
enabled.

Signed-off-by: laiguo-liang <2909269677@qq.com>
---
 net/core/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/core/Makefile b/net/core/Makefile
--- a/net/core/Makefile
+++ b/net/core/Makefile
@@ -36,6 +36,7 @@
 obj-$(CONFIG_HWBM) += hwbm.o
 obj-$(CONFIG_GRO_CELLS) += gro_cells.o
 obj-$(CONFIG_FAILOVER) += failover.o
+obj-$(CONFIG_NET_DELAYACCT) += net-delayacct.o
 obj-$(CONFIG_NET_SOCK_MSG) += skmsg.o
 obj-$(CONFIG_BPF_SYSCALL) += sock_map.o
 obj-$(CONFIG_BPF_SYSCALL) += bpf_sk_storage.o
